conditional-al-pn.js

```diff
@@ -0,0 +1,14 @@
+//Write a statement that takes a variable of item and logs "in budget" if a price is $100 or less.
+var item = 5
+if (item >= 100){
```

Comment:
This setup is great. Just be mindful of the relational operator. The equal sign = is needed. Less than should be <